mne/viz/topomap.py

```diff
@@ -1120,12 +1120,18 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
     mask_params['markersize'] *= size / 2.
     mask_params['markeredgewidth'] *= size / 2.
 
+    picks, pos, merge_grads, names, ch_type = _prepare_topo_plot(
+        evoked, ch_type, layout)
+
+    evoked_ = evoked.pick_channels([evoked.ch_names[pick] for pick in picks],
+                                   copy=True)
```

Comment:
below in the same function there is a 

evoked.copy().apply_proj().data

I suspect it's problematic.

to me you should first apply projs, then pick to get the data